frontend/src/pages/journal/components/note/components/note-input/note-input.tsx

```diff
@@ -0,0 +1,86 @@
+import { useCallback, useFormController } from '#libs/hooks/hooks.js';
+import {
+  type FormControl,
+  type FormFieldPath,
+  type FormFieldValues,
+} from '#libs/types/types.js';
+
+import {
+  ContentEditable,
+  LexicalComposer,
+  LexicalErrorBoundary,
+  PlainTextPlugin,
+} from './components/components.js';
+import {
+  $createParagraphNode,
+  $createTextNode,
+  $getRoot,
+} from './libs/helpers/helpers.js';
+import { OnChangePlugin } from './libs/plugins/plugins.js';
+import {
+  type ErrorBoundaryType,
+  type LexicalEditor,
+} from './libs/types/types.js';
+import styles from './styles.module.scss';
+
+type Properties<T extends FormFieldValues> = {
+  name: FormFieldPath<T>;
+  control: FormControl<T, null>;
+  placeholder: string;
+  contentEditableStyle?: string | undefined;
+  onError: (error: Error) => void;
+};
+
+const NoteInput = <T extends FormFieldValues>({
+  placeholder,
+  name,
+  control,
+  contentEditableStyle,
+  onError,
+}: Properties<T>): JSX.Element => {
+  const { field } = useFormController({ name, control });
+  const { value, onChange } = field;
+
+  const handleChange = useCallback(
+    (newValue: string) => {
+      const parser = new DOMParser();
+      const dom = parser.parseFromString(newValue, 'text/html');
+
+      onChange(dom.body.textContent);
+    },
+    [onChange],
+  );
```

Comment:
it will also make the note body into a simple text. It removes the whole sense of the editor then

Reply:
Line break is working now) We need to merge this PR ASAP, so current scope doesn't contain advanced editor

Reply:
Yep, I agree with @HIMDanny . Now logic with the new line doesn't work as expected. As it converts to simple text, it doesn't know anything about indentation after page reloading.

Reply:
@v0ldemar01 As I can see in my localhost, it works when you are typing, but after reloading you can see one line. Do you have another behavior?

Reply:
I don't think it works properly now)

How I typed:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/58943798/00e16d92-e0b9-4f40-ad43-005c25fed586)

After reload:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/58943798/5fb31acd-477b-490e-95c6-9899ac7e79fc)

Reply:
Yeah, I told above about this.

Reply:
Got it)